retis-events/src/helpers.rs

```diff
@@ -52,6 +54,27 @@ pub(crate) fn protocol_str(protocol: u8) -> Option<&'static str> {
     })
 }
 
+/// u128 representation in the events. We can't use the Rust primitive as serde
+/// does not handle the type well.
+#[event_type]
+pub struct U128 {
+    hi: u64,
+    lo: u64,
+}
```

Comment:
Due to serde limitations I chose to represent `u128` as the above. Another way would be to use something like `U128(u128)` and represent it as a string in JSON.